src/SiteMaster/Core/Auditor/Logger/Links.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace SiteMaster\Core\Auditor\Logger;
+
+use DOMXPath;
+use SiteMaster\Core\Auditor\Site\Page;
+use SiteMaster\Core\Util;
+
+class Links extends \Spider_LoggerAbstract
+{
+    /**
+     * @var bool|\Spider
+     */
+    protected $spider = false;
```

Comment:
Is it common practice to use false here? A null pointer (uninitialized) seems it would be fine. Setting it to a boolean make me think there should be some sort of error handling in this class.